content/guides/outliers.md

```diff
@@ -80,75 +80,51 @@ There are two different outlier detection algorithms you can use on your data: D
 
-The following graphs show a set of hosts where there are several distinct groups. A low setting of alpha marks both the host handling significantly less traffic and the group of hosts that are handling more traffic than the majority as outliers.
+The  [Median Absolute Deviation](https://en.wikipedia.org/wiki/Median_absolute_deviation) is a robust measure of variability, and can be viewed as the robust analog for standard deviation. Robust statistics describe data in such a way that they are not unduly influenced by outliers.
 
-<img src="/static/images/outliers/outliers-dbscan-banding.png" style="width:100%; border:1px solid #777777"/>
+For a given set of data D = {d<sub>1</sub>, ..., d<sub>n</sub>}, the deviations are the difference between each d<sub>i</sub> and median(D). The MAD is then the median of the absolute values of all the deviations. For example if D = {1, 2, 3, 4, 5, 6, 100}, then the median is 4, the deviations are {-3, -2, -1, 0, 1, 2, 96}, and the MAD is 2. (Note that the standard deviation by contrast is 33.8.)
 
-Meanwhile a higher setting of alpha only identifies the host handling significantly less traffic as an outlier.
+#### Parameters
 
-<img src="/static/images/outliers/outliers-dbscan-no-banding.png" style="width:100%; border:1px solid #777777"/>
+In our case, the data set is the set of all points in every time series. We take the MAD of all the points then multiply it by 1.4826 and our first parameter, **scale**. The first constant normalizes MAD so that it is comparable to the standard deviation of the normal distribution. The scale parameter then specifies how many “deviations” a point has to be away from the median for it to be considered an outlier.
 
-### Median Absolute Deviation (MAD)
+Now to mark a time series as an outlier, we use the second parameter, **pct**. If more than pct% of a particular series’ points are considered outliers, then the whole series is marked to be an outlier. Here is MAD with a scale of 3 and pct of 20 in action when comparing the average system load by availability zone:
 
-The [Median Absolute Deviation](https://en.wikipedia.org/wiki/Median_absolute_deviation) (MAD) algorithm is a robust method of calculating variability among data. It is designed to be usable for both normal and non-normal distributions.
+<img src="/static/images/outliers/outliers-mad-az.png" style="width:100%; border:1px solid #777777"/>
 
-We use MAD to calculate an *outlier_factor* which we will compare to points from a series. If the percentage of points in the series that are above the *outlier_factor* is greater than some given threshold, then that series will be considered an outlier.
+The scale parameter should be tuned depending on expected variability of the data. For example if data is generally within a small range of values then this should be small. On the other hand if points can vary greatly then you want a higher scale so these variabilities do not trigger a false positive.
```

Comment:
Missing commas after "For example," "range of values," "On the other hand," "vary greatly."